core/runtime/src/main/java/io/quarkiverse/langchain4j/guardrails/GuardrailsUtils.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkiverse.langchain4j.guardrails;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+class GuardrailsUtils {
+    private GuardrailsUtils() {
+    }
+
+    private static class MapperHolder {
+        private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Another mapper? As I sid, we should avoid instantiating these.

Reply:
I will try to replace it with the one retrieved through Quarkus injection.

Reply:
You can use programmatic lookup and only create one if none are available.